packages/modules/src/SessionKeyManagerModule.ts

```diff
@@ -7,7 +7,6 @@ import {
   SessionKeyManagerModuleConfig,
   ModuleVersion,
   CreateSessionDataParams,
-  StorageType,
```

Comment:
can we not remove this storageType please

Reply:
Fixed